pkg/generated/restapi/configure_rekor_server.go

```diff
@@ -174,21 +178,81 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 	return handler
 }
 
+type httpRequestFields struct {
+	requestMethod string
+	requestURL    string
+	requestSize   string
+	status        int
+	responseSize  string
+	userAgent     string
+	remoteIP      string
+	latency       string
+	protocol      string
+}
+
+func (h httpRequestFields) MarshalLogObject(enc zapcore.ObjectEncoder) error {
+	enc.AddString("requestMethod", h.requestMethod)
+	enc.AddString("requestUrl", h.requestURL)
+	enc.AddString("requestSize", h.requestSize)
+	enc.AddInt("status", h.status)
+	enc.AddString("responseSize", h.responseSize)
+	enc.AddString("userAgent", h.userAgent)
+	enc.AddString("remoteIP", h.remoteIP)
+	enc.AddString("latency", h.latency)
```

Comment:
enc.AddDuration("latency", h.latency)

Reply:
https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest wants this represented as a string in units of seconds, so that's why I formatted it the way I did